controllers/ovndbcluster_controller.go

```diff
@@ -612,6 +625,16 @@ func (r *OVNDBClusterReconciler) reconcileNormal(ctx context.Context, instance *
 
 		// Set DB Address
 		instance.Status.InternalDBAddress = strings.Join(internalDbAddress, ",")
+		if instance.Spec.DBType == ovnv1.SBDBType && instance.Spec.NetworkAttachment != "" {
```

Comment:
what's the reason this is embedded under ReadyCount check? I think we should create the ConfigMap regardless of the current state of db pods.

Reply:
It was done to try to mimic what OVNController was doing. Also it was to ensure that when the configMap was created, environment was already ready to be used by EDPM nodes.
Also, given the time difference between creating configMap earlier on the reconcile loop or here would/should be minimal (order of units of seconds?) which is negligible compared with the EDPM deployment time. I think it's better to create configMap once pods are ready in order to catch the error earlier (EDPM deployment failed due to configMap not existing vs edpm not working correctly bc can't connect to SB).
Let me know what you think

Reply:
I kinda disagree (one should be able to deploy EDP regardless of the current state of the SB service - otherwise a hiccup in SB may break the EDP deployment; we may want to make sure that ovn-controller on EDP starts successfully even without SB, but I think it should already do that - it should gracefully reconnect.)

I don't insist.